lib/state/log.go

```diff
@@ -0,0 +1,108 @@
+package state
+
+import (
+	"io"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/gravitational/teleport/lib/events"
+	"github.com/gravitational/teleport/lib/session"
+	"github.com/gravitational/trace"
+)
+
+var (
+	errNotSupported = trace.Errorf("method not supported")
```

Comment:
let's not use `trace.Errorf` as we agreed, you can return `trace.BadParameter("method not supported")` instead.